### PR TITLE
3D tiles layer sample

### PR DESCRIPTION
**Pull request by TADraeseke**
## Description

PR to add a new Kotlin sample "Add 3d tile layer" in "Layers category.

## Links and Data

https://devtopia.esri.com/runtime/kotlin/issues/3845

## What To Review

-  Review the code to make sure it is easy to follow like other samples on Android
- `README.md` and `README.metadata.json` files

## How to Test

Run the sample on the sample viewer or the repo.

### CODE REVIEW

File: add-3d-tiles-layer/README.metadata.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "Camera",
+        "Ogc3DTilesLayer",
+        "SceneView"
+    ],
+    "snippets": [
+        "src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/MainActivity.kt",
+        "src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/components/SceneViewModel.kt",
+        "src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/screens/MainScreen.kt"
+    ],
+    "title": "Add 3d tiles layer"
```
Comment (alan-edi) [inline add-3d-tiles-layer/README.metadata.json:33]: 3d -> 3D ?

File: add-3d-tiles-layer/src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+        setContent {
+            SampleAppTheme {
+                Add3dTilesLayerApp()
+            }
+        }
+    }
+
+    @Composable
+    private fun Add3dTilesLayerApp() {
```
Comment (alan-edi) [inline add-3d-tiles-layer/src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/MainActivity.kt:46]: 3d -> 3D ?

File: add-3d-tiles-layer/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">Add 3d tiles layer</string>
```
Comment (alan-edi) [inline add-3d-tiles-layer/src/main/res/values/strings.xml:2]: 3d -> 3D ?

**Review by alan-edi — APPROVED** [review]
@TADraeseke LGTM. Just one suggestion (repeated 3 times).

File: add-3d-tiles-layer/src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/components/SceneViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+import com.arcgismaps.mapping.ArcGISTiledElevationSource
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.layers.Ogc3DTilesLayer
+import com.arcgismaps.mapping.view.Camera
+import com.arcgismaps.toolkit.geoviewcompose.SceneViewProxy
+
+class SceneViewModel(application: Application) : AndroidViewModel(application) {
+
+    // Create a SceneViewProxy that gets passed to the SceneView composable in MainScreen.kt.
```
Comment (shubham7109) [inline add-3d-tiles-layer/src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/components/SceneViewModel.kt:31]:
```suggestion
    // Create a SceneViewProxy which is passed to the composable SceneView
```

File: add-3d-tiles-layer/src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/components/SceneViewModel.kt [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    val sceneViewProxy = SceneViewProxy()
+
+    // Create a scene with a dark gray basemap. The scene is added to the composable SceneView in
+    // MainScreen.kt.
```
Comment (shubham7109) [inline add-3d-tiles-layer/src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/components/SceneViewModel.kt:35]:
```suggestion
    // Create a scene with a dark gray basemap that gets passed to the composable SceneView.
```

File: add-3d-tiles-layer/src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/components/SceneViewModel.kt [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        // buildings in Stuttgart, Germany.
+        operationalLayers.add(
+            Ogc3DTilesLayer(
+                "https://tiles.arcgis.com/tiles/ZQgQTuoyBrtmoGdP/arcgis/rest/services/Stuttgart/3DTilesServer/tileset.json"
+            )
+        )
+
+        // Set the initial viewpoint of the scene. The viewpoint is set to a location in Stuttgart.
+        initialViewpoint = Viewpoint(
+            48.8466, 9.1627, 1000.0, Camera(
```
Comment (shubham7109) [inline add-3d-tiles-layer/src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/components/SceneViewModel.kt:55]: Named args would be helpful here for readability. 
```suggestion
            latitude = 48.8466, longitude = 9.1627, scale = 1000.0,
            camera = Camera(
```

File: add-3d-tiles-layer/src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/screens/MainScreen.kt [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+import com.arcgismaps.toolkit.geoviewcompose.SceneView
+import com.esri.arcgismaps.sample.add3dtileslayer.components.SceneViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun MainScreen(sampleName: String) {
+    // create a ViewModel to handle MapView interactions
```
Comment (shubham7109) [inline add-3d-tiles-layer/src/main/java/com/esri/arcgismaps/sample/add3dtileslayer/screens/MainScreen.kt:34]:
```suggestion
    // create a ViewModel to handle SceneView interactions
```

**Review by shubham7109 — APPROVED** [review]
Looks good @TADraeseke! Just few minor suggestions.

**Pull request merged**